### PR TITLE
Release v0.x

### PR DESCRIPTION
**Pull request by juliomoreira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+
+or
+
+Run the following command:
+
+```sh
+vtex install vtex.criteo@0.x
+```
+
+Next, open the app settings on your admin and fill the form with you Criteo details.
```
Comment (klzns) [inline README.md]: *Your

File: pixel/head.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    if (criteo_id) {
+      var a = document.createElement("script");
+      a.type = "text/javascript";
+      a.async = true;
+      a.src = "//static.criteo.net/js/ld/ld.js";
+      var s = document.getElementsByTagName("script")[0];
+      s.parentNode.insertBefore(a, s);
+    }
+    window.criteo_id = criteo_id;
+  })(window, window.criteo_id);
```
Comment (klzns) [inline pixel/head.html]: You can remove this second param

File: pixel/head.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<script>
+  (function(window, criteo_id) {
```
Comment (klzns) [inline pixel/head.html]: Same here

File: react/index.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
+import { canUseDOM } from 'vtex.render-runtime'
+import { PixelMessage } from './typings/events'
+import { fetchEmail, getSiteType } from './helpers'
+
+function impression(event: PixelMessage) {
+  switch (event.data.eventName) {
+    case 'vtex:pageInfo':
```
Comment (klzns) [inline react/index.tsx:7]: Pageinfo is triggered in other pages, not just in home page

File: react/index.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+      return { event: 'viewHome' }
+    case 'vtex:categoryView':
+      return {
+        event: 'viewList',
+        item: event.data.products
+          .map(({ productId }: any) => productId)
+          .slice(0, 3),
```
Comment (klzns) [inline react/index.tsx]: Slice first, so we map in less items

File: react/index.tsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+      }
+    case 'vtex:productView':
+      const {
+        data: {
+          product: { productId: item },
+        },
+      } = event.data.product.productId
```
Comment (klzns) [inline react/index.tsx]: Is that right?
Should this be just event?

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/index.tsx (hunk 4 above)
Comment (juliomoreira) [reply react/index.tsx:7]: Added the raw `eventType` as a fallback just in case..

**Review by juliomoreira — COMMENTED** [review]
@klzns reviewed, can you check again please?

**Review by klzns — APPROVED** [review]

**Pull request merged**